documentation/modules/compatibility-guidelines.adoc

```diff
@@ -18,8 +18,8 @@ You must install compatible software versions.
 [NOTE]
 .Migration from {rhv-full} 4.3
 ====
-MTV 2.5 was tested only with {rhv-full} (RHV) 4.4 SP1.
-Migration from {rhv-full} ({rhv-short}) 4.3 has not been tested with {project-short} {project-version}.
+{project-short} 2.6 was tested only with {rhv-full} (RHV) 4.4 SP1.
```

Comment:
it's very unlikely that we'll test migrations from RHV 4.3 in the future and as there's no plan to release another version of RHV, how about replacing `2.6` with `{project-version}` so that we won't need to update it each time?